src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -44,19 +44,12 @@ public Order placeOrder(User user, ShippingAddressRequestDto shippingAddress) {
                 .mapToDouble(cartItem -> (double) cartItem.getQuantity()
                         * cartItem.getBook().getPrice().doubleValue())
                 .sum();
-        Order order = new Order();
-        order.setUser(user);
-        order.setShippingAddress(shippingAddress.getShippingAddress());
-        order.setOrderDate(LocalDateTime.now());
-        order.setStatus(Order.Status.NEW);
-        order.setTotal(BigDecimal.valueOf(total));
+        OrderRequestDto orderRequestDto = orderMapper.toDto(shoppingCart);
+        Order order = orderMapper.toModel(orderRequestDto);
```

Comment:
I cannot see reason why shopping cart should be mapped to OrderRequestDto to instantly remapped to Order

just make constructor for Order that accepts ShoppingCart
```suggestion
        Order order = new Order(shoppingCart);
```